src/test/java/com/segment/analytics/android/integration/firebase/FirebaseTest.java

```diff
@@ -111,13 +107,42 @@ public void trackWithProperties() {
         expected.putString("string", "foo");
         expected.putString("date", String.valueOf(new Date(117, 0, 1)));
         expected.putString("key_with_spaces", "bar");
+        expected.putString("key_with_periods", "test");
         expected.putDouble("value", 100.0);
         expected.putString("currency", "USD");
         expected.putString("extra_spaces", "baz");
 
         verify(firebase).logEvent(eq("foo"), bundleEq(expected));
     }
 
+    @Test
```

Comment:
The spacing looks like it _may_ be a little off here. Running `./gradlew fmt` should automatically fix spacing and formatting issues for you.

Reply:
@brennan ran the command but not changes were applied to commit.